### PR TITLE
Website: Knowledge Base

### PR DESCRIPTION
**Pull request by figitaki**
Implements the knowledge base.

One caveat: **NO UNPUBLISHED RESOURCES** if you add a "Knowledge Base Resource" in *Draft* to a "Knowledge Base Category" it will crash the site.

A little hairy in the Contentful integration, but lmk if it needs more comments or refactoring. Might need to read this for context: https://www.contentful.com/developers/docs/concepts/links/

### CODE REVIEW

File: frontend/website-redesign/src/pages/Tech.re [hunk 1]
```diff
@@ -185,7 +184,7 @@ module HowMinaWorks = {
           </b>
           {React.string(
              {js|
-            When anyone can enforce the rules by validating an irrevocable public
+            WheknowledgeBaseCategoryn anyone can enforce the rules by validating an irrevocable public
```
Comment (garethtdavies) [inline frontend/website-redesign/src/pages/Tech.re:187]: Looks like this inadvertently slipped in

Comment (figitaki) [reply frontend/website-redesign/src/pages/Tech.re:187]: Great catch, thanks.

**Review by MartinMinkov — CHANGES_REQUESTED** [review]
Could you add the Knowledgebase to the other pages that need it as well? 

I left a `TODO` comment in NodeOperators, it'd be awesome if you could throw it in! I think the getting started page also needs it from inspection of the designs.

**Comment by michellewong793** [comment]
mobile photo here looks like its the wrong size 
image


ipad photo too

**Review by michellewong793 — APPROVED** [review]

**Pull request merged**